OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -237,7 +262,18 @@ function Import-ServerConfig {
 
     if (Test-OctopusVersionSupportsShowConfiguration) {
         $rawConfig = & $octopusServerExePath show-configuration --format=json-hierarchical --noconsolelogging --console --instance $InstanceName
-        $config = $rawConfig | ConvertFrom-Json
+
```

Comment:
Might be worth sticking a pester test around this

Reply:
Added a couple of simple pester tests.